dds/DCPS/DiscoveryBase.h

```diff
@@ -1163,9 +1176,19 @@ namespace OpenDDS {
 
 
       struct MatchingData {
-        RepoId writer;
-        RepoId reader;
-        SequenceNumber rpc_sequence_number;
+        // Sequence number of the first request for remote minimal types.
+        SequenceNumber rpc_seqnum_minimal;
+
+        // Whether all minimal types are obtained.
+        bool got_minimal;
```

Comment:
Prior to this change, `MatchingData` didn't have any data members that needed initialization beyond the defaults.  Now that there are the two `bool` members, there should be an explicit default constructor and not just the compiler generated one.